src/components/mobile.css

```diff
@@ -163,12 +163,11 @@
     width: auto;
     height: auto;
     padding: auto;
-    margin: auto;
     overflow: visible;
     clip: none;
     border: auto;
     line-height: 1.2;
-    margin-top: 4.2rem;
+    margin: 4.2rem auto auto;
```

Comment:
question: How does this help? `margin-top` explicitly only sets the top margin, whereas now we're setting the right, bottom and left margins too.

Reply:
Lets explicitly set them to `0` if we want them to be zero – I like the more explicit callout of not using `margin-top` myself 👍